neo/Network/RPC/RpcServer.cs

```diff
@@ -81,7 +81,7 @@ public RpcServer(NeoSystem system, Wallet wallet = null, long maxGasInvoke = def
             this.MaxGasInvoke = maxGasInvoke;
         }
 
-        private static JObject CreateErrorResponse(JObject id, int code, string message, JObject data = null)
+        internal static JObject CreateErrorResponse(JObject id, int code, string message, JObject data = null)
```

Comment:
Why `internal`?

Reply:
This method is covered in unit test.

Reply:
https://github.com/neo-project/neo/pull/943#discussion_r306254478

According to the discussion here, we don't need to test private methods. Testing for public methods can completely cover private methods.